microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -113,123 +129,60 @@ keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft S
     :::column-end:::
 :::row-end:::
 
-Build \(or convert\) your existing website to a PWA to enhance engagement with your users. Enhancements include push notifications, app-like integration, and offline support. Continue to build your audience on the open web for users to discover your PWA through search and link-sharing. Best of all, your app is updated using your web server code.
+## PWAs on desktop
 
-## PWAs on Microsoft Edge (Chromium)
+PWAs aren't only a way for websites to appear on mobile home screens. They're based on standard, cross-browser, web technologies that allow them to be installed and to run everywhere.
 
-When you build a Progressive Web App targeting web standard APIs, your app may be deployed across platforms and devices and take advantage of the device-specific capabilities as available. PWAs in Microsoft Edge \(Chromium\) add the following advantages to your website.
+While most consumers have migrated to mobile-first ways of consuming content, businesses still mostly operate on desktop.
+Building your desktop app as a PWA allows you to make use of all the benefits that PWAs come with while targeting those users.
 
-*   Your app is built on a standards-based web platform.
-*   Allows your users to install your app directly from the browser.
-*   Allows your users to install your app without a Store-based deployment or registration.
+With many new web capabilities and new features in Microsoft Edge and Windows, there are great opportunities to build immersive experiences that deeply integrate with the desktop operating system and feel native. Only using web technologies also ensures that your app will work in other browsers, operating systems and devices.
 
-Desktop PWAs are supported on any of the platforms Microsoft Edge \(Chromium\) is [available](https://www.microsoft.com/edge). The following benefits are included.
+You might not have to use native technologies at all as the web has grown capabilities to deal with file systems, video, audio, high-performing code, databases, USB, Bluetooth, and more.
 
-*   Apps may be installed directly from within the browser using the **Install** icon in the navigation bar.
+With a shared codebase between your website, mobile app and desktop app (across operating systems), you cut down on development cost.
```

Comment:
What’s our position on the Oxford comma? (I always forget.)

Reply:
https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#remember-the-last-comma
Remember the last comma
In a list of three or more items, include a comma before the conjunction. (The comma that comes before the conjunction is known as the Oxford or serial comma.) To learn more, see Commas.  https://docs.microsoft.com/en-us/style-guide/punctuation/commas

Example
Replace this: Android, iOS and Windows

With this: Android, iOS, and Windows